src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,28 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
```

Comment:
Please include a header comment to indicate what this interface/class/method does?